ldb/models.py

```diff
@@ -410,7 +410,8 @@ class Meta:
     first_year = models.IntegerField(_('first year'), blank=True, null=True)
     student_number = CharNullField(_('student number'), max_length=7, blank=True, null=True, unique=True)
 
-    phone_parents = models.CharField(_('phone parents'), max_length=16, blank=True)
+    emergency_name = models.CharField(_('emergency name'), max_length=16, blank=True)
+    emergency_phone = models.CharField(_('emergency phone'), max_length=16, blank=True)
```

Comment:
Some phone numbers may be longer than 16 chars, right? With international students having country codes in their phone numbers etc. Also, a max length of 16 for a name, shouldn't this be more?

Reply:
No, the maximum is 15 digits: https://en.wikipedia.org/wiki/MSISDN

Reply:
Alright. I will update the emergency name to something bigger then. Say 48 chars.